### DISCUSSION TITLE
Sleeper and alternatives

### DISCUSSION THREAD

**Discussion by ahwagner** [GenomicMedLab/fantasy-football-rulebook#6] [General]
In #4, we voted to try out the Sleeper platform. After 1 season on the new platform, we identified some challenges:

- Enrollment using devices other than mobile was difficult (@malachig)
- Less support in Sleeper for pre-draft queuing ([Sleeper support](https://support.sleeper.com/en/articles/4268342-can-i-set-pre-draft-rankings) v. [ESPN support](https://support.espn.com/hc/en-us/articles/360000139672-Updating-Your-Draft-List)) (@obigriffith)
- Using the Sleeper platform to manage team (@susannasiebert, @tmooney, @LMT337)

This thread is for discussion (and ultimately voting) on whether or not to revert to the ESPN platform (or try another platform) for the 2024 season.

**Comment by katie-perry** [comment]
I've used Yahoo, ESPN, and Sleeper now for fantasy leagues. I think Yahoo was my favorite as far as ease-of-use and helpful features. But, given a choice between ESPN and Sleeper, I would vote ESPN. 
My reasoning for preferring ESPN to Sleeper is that I found the user interface to be more friendly and intuitive. I found it much easier to view reporting for player outcomes, projections, and statistics. Additionally, reporting seemed more accurate from ESPN than Sleeper. (Too many times, Sleeper would report a player to be healthy and they'd be out, meanwhile ESPN was clearer with reporting that a player would be unlikely to play... this probably comes down to an analyst difference between the companies... but still)

Out of the 3, Yahoo edged out the others because I liked that it had more options than ESPN and Sleeper for filtering players, and player/team projections seemed to update more regularly and were more accurate. I don't know anything about how the draft process is, though, since the year I used this league, we did our draft on an excel spreadsheet and the commissioner entered everyone's teams manually... 

Unless anyone else joins in singing praises about Yahoo, I am a proponent of switching back to ESPN

**Comment by tmooney** [comment]
I agree that Sleeper's analysis seemed less accurate than ESPN's seemed to be in previous seasons. OTOH considering I do no outside research beyond what the app shares, it clearly was good enough for my needs. 🙂 I definitely did have to be more vigilant to double-check lineups Sunday morning.

I do mostly like the Sleeper interface now that I've mostly figured it out--it has some handy features that ESPN didn't, but I'd also be fine with moving back to ESPN as it's pretty familiar.  I'm a little reluctant to keep switching platforms every year!  My initial vote last year for not switching to Sleeper was mostly predicated on being fine with the system we had.  I'm also fine with the system we currently have, so either way!
